### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Grid

## Trello Card
https://trello.com/c/NqgITw42/12-redise%C3%B1o-de-la-p%C3%A1gina
## Screenshots
captura de pantalla 2018-10-29 a la s 14 49 17

### CODE REVIEW

File: maquetado/scss/tech.scss [hunk 1]
```diff
@@ -20,17 +20,16 @@
 }
 
 .tech-container {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  align-self: center;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: I think that you don't need this, use it in the `first-child` definition

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -20,17 +20,16 @@
...
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: I think that you don't need this

File: maquetado/scss/tech.scss [hunk 3]
```diff
@@ -20,17 +20,16 @@
 }
 
 .tech-container {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  align-self: center;
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Change this for `grid-template-columns: repeat(3, 1fr);`

File: maquetado/scss/tech.scss [hunk 4]
```diff
@@ -39,20 +38,11 @@
...
-    height: 100%;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    transition: transform 0.5s ease;
-    width: 100%;
-    z-index: -1;
+  &:first-child {
+    border-right: 2px solid $green;
+    grid-area: 1/1/3/2;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: You can change this for something more simple as `grid-row: span 2`

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good job

File: maquetado/scss/tech.scss [hunk 5]
```diff
@@ -39,20 +36,12 @@
...
-    top: -10px;
-    transition: transform 0.5s ease;
-    width: 100%;
-    z-index: -1;
+  &:first-child {
+    align-self: center;
+    border-right: 2px solid $green;
+    grid-row: span 2;
+    justify-content: center;
+    padding: 0 100px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss:44]: Do you need this padding?

**Review by henryzarza — APPROVED** [review]

File: maquetado/scss/styles.scss [hunk 6]
```diff
@@ -1,5 +1,9 @@
 .content {
-  padding: 80px 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
```
Comment (SKOLZ) [inline maquetado/scss/styles.scss]: this is the default value so it's not needed

File: maquetado/scss/styles.scss [hunk 7]
```diff
@@ -1,5 +1,9 @@
...
+  justify-content: center;
```
Comment (SKOLZ) [inline maquetado/scss/styles.scss]: hmmm I think this is not needed, why have you added this.

File: maquetado/scss/tech.scss [hunk 8]
```diff
@@ -39,20 +36,12 @@
...
+    width: auto;
```
Comment (SKOLZ) [inline maquetado/scss/tech.scss]: this is the default value, you can delete it.

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**